src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,87 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
+  amount: string
+  symbol: string
+  discountPercent: string
+  isClaimed?: boolean
+  onClaim?: () => void
+}
+
+const actionsPaddingLeft = { base: 10, md: 0 }
+const columnWidth = { base: '100%', md: '50%' }
+const columnDirection: StackDirection = { base: 'column', md: 'row' }
+const columnAlignItems = { md: 'center' }
+const columnJustifyContent = { md: 'space-between' }
+const columnSpacing = { base: 4, md: 48 }
+
+export const FoxWifHatHoldingLine = ({
+  accountId,
+  accountNumber,
+  amount,
+  symbol,
+  discountPercent,
+  isClaimed,
+  onClaim,
+}: FoxWifHatHoldingLineProps) => {
+  const textColor = useColorModeValue('gray.500', 'gray.400')
+  const translate = useTranslate()
+
+  return (
+    <Stack
+      width='full'
+      px={6}
+      py={4}
+      spacing={columnSpacing}
+      direction={columnDirection}
+      alignItems={columnAlignItems}
+      justifyContent={columnJustifyContent}
+    >
+      <Flex width={columnWidth} alignItems='center' justifyContent='space-between'>
+        <HStack spacing={4}>
+          <WalletIcon color={textColor} boxSize={6} />
+          <Flex direction='column' alignItems='flex-start'>
+            <Text fontWeight='bold' fontSize='sm'>
+              {middleEllipsis(accountId)}
+            </Text>
+            <Text color={textColor} fontSize='xs'>
+              {translate('accounts.accountNumber', { accountNumber })}
+            </Text>
+          </Flex>
+        </HStack>
+
+        <Text fontWeight='bold'>
+          {amount} {symbol}
+        </Text>
+      </Flex>
+
+      <Flex
+        width={columnWidth}
+        alignItems='center'
+        justifyContent='space-between'
+        pl={actionsPaddingLeft}
+      >
+        <Text color='green.500' fontSize='sm' fontWeight='bold'>
+          {translate('foxPage.foxWifHat.discountText', { percent: discountPercent })}
```

Comment:
suggestion: if going with `percentDecimal` vernacular, can use `toPercent()` which ensures consistent formatting

Reply:
Done in [a79f134](https://github.com/shapeshift/web/pull/8605/commits/a79f1341343395bca74b469a901a3a4e12a71be0)